bitcoin/script/miniscript.h

```diff
@@ -397,10 +397,19 @@ struct Node {
     //! Internal code for ToString.
     template<typename Ctx>
     std::string MakeString(const Ctx& ctx, bool& success, bool wrapped = false) const {
+        std::string ret = wrapped ? ":" : "";
```

Comment:
It looks like this would output multi-colon when there are multiple wrappers. Currently, this is not valid in rust-miniscript, we should probably have some consensus on this.